heartbeat/sender/peerAuthenticationSender.go

```diff
@@ -170,6 +170,9 @@ func (sender *peerAuthenticationSender) execute() (error, bool) {
 		return err, isTriggered
 	}
 
+	log.Debug("sending peer authentication message",
```

Comment:
Do we keep this on Debug level?

Reply:
yes, one message at ~2hours. Helps the debugging a lot.